README-GENERATOR.md

```diff
@@ -100,6 +100,10 @@ pagination:
   # Without file extension
   indexpage: 'index'
 
+  # Optional, the default page values of the generated pages (e.g. 'sitemap: false')
+  # Without file extension
+  default_values: nul
```

Comment:
`nul` is not the correct usage in English.
It should either be `null`, or `~`, or simply ` `

Reply:
You're right I'll fix that with

```suggestion
  default_values: null
```